package.json

```diff
@@ -26,14 +26,15 @@
   },
   "homepage": "https://github.com/scality/bucketclient#readme",
   "devDependencies": {
-    "mocha": "",
     "eslint": "^2.4.0",
     "eslint-config-airbnb": "^6.0.0",
     "eslint-config-scality": "scality/Guidelines#71a059ad",
-    "eslint-plugin-react": "4.2.3"
+    "eslint-plugin-react": "4.2.3",
+    "mocha": ""
```

Comment:
What's reason to remove mocha version?

Reply:
nvm, for a sec I thought it was new, just realized it's just on new line.

Reply:
yes, yarn did this on its own actually